docs/sources/datasources/tempo/query-editor/_index.md

```diff
@@ -26,86 +26,167 @@ refs:
-Both of these methods let you build queries and drill-down into result sets.
+You don't have to know TraceQL to create a query.
+You can use the Search query builder's user interface to select options to search your data.
+These selections generate a TraceQL query.
+Any query generated in Build mode can be transferred to the Code mode, where you can edit the query directly.
 
 To learn more about how to query by TraceQL, refer to the [TraceQL documentation](/docs/tempo/latest/traceql).
 
-### TraceQL query builder
+## Choose a query editing mode
 
-The TraceQL query builder, located on the **Explore** > **Query type** > **Search** in Grafana, provides drop-downs and text fields to help you write a query.
+The query editor has three modes, or **Query types**, that you can use to explore your tracing data.
+You can use these modes by themselves or in combination to create building blocks to generate custom queries.
+
+![The three query types: Search, TraceQL, and Service Graph](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-types.png)
+
+The three **Query types** are:
+
+- The **Search** query builder provides a user interface for building a TraceQL query.
+- The **TraceQL** query editor lets you write your own TraceQL query with assistance from autocomplete.
+- The **Service Graph** view displays a visual relationship between services. Refer to the [Service graph](ref:service-graph) documentation for more information.
+
+### Search query builder
+
+The Search query builder provides drop-down lists and text fields to help you write a query.
+The query builder is ideal for people who aren't familiar with or want to learn TraceQL.
 
 Refer to the [Search using the TraceQL query builder documentation]({{< relref "./traceql-search" >}}) to learn more about creating queries using convenient drop-down menus.
 
-![The TraceQL query builder](/static/img/docs/tempo/screenshot-traceql-query-type-search-v10.png)
+![The Search query builder](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-search-v11.png)
 
-### TraceQL query editor
+### TraceQL Query editor
 
-The TraceQL query editor, located on the **Explore** > **TraceQL** tab in Grafana, lets you search by trace ID and write TraceQL queries using autocomplete.
+The TraceQL query editor lets you search by trace ID and write TraceQL queries using autocomplete.
 
 Refer to the [TraceQL query editor documentation]({{< relref "./traceql-editor" >}}) to learn more about constructing queries using a code-editor-like experience.
 
-![The TraceQL query editor](/static/img/docs/tempo/screenshot-traceql-query-editor-v10.png)
+![The TraceQL query editor](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-traceql-v11.png)
+
+You can also search for a Trace ID by entering a trace ID into the query field.
+
+### Service graph view
+
+Grafana’s service graph view uses metrics to display span request rates, error rates, and durations, as well as service graphs.
```

Comment:
To keep this consistent, gonna format this like the others.
```suggestion
Grafana’s **Service Graph** view uses metrics to display span request rates, error rates, and durations, as well as service graphs.
```

Reply:
Thank you. So good to have a fresh set of eyes on this.